packages/ui/src/theme/Color.ts

```diff
@@ -336,7 +179,8 @@ export enum ColorDarkInternal {
   // ???
   //
 
-  White = '#fff',
+  White = 'rgba(255, 255, 255, 0.5)',
+  DynamicWhite = 'rgba(25, 35, 52, 1)',
```

Comment:
@MekhrangizY **nitpick**: if it's do not have opacity, maybe we can use hex value instead?

Reply:
Artyom told to put it in rgba -  DynamicWhite = 'rgba(25, 35, 52, 1)',
for white sure i will change to hex